### PR TITLE
Initial implementtation

### PR DESCRIPTION
**Pull request by neumantm**

### CODE REVIEW

File: mail_alias_manager/db/models/receiving_alias.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.sql.schema import Column
+from sqlalchemy_utils.types.email import EmailType
+
+from ..db import DB, MODEL
+
+
+class ReceivingAlias(MODEL):
+    id: Column = DB.Column(DB.Integer, primary_key=True)
+    alias: Column = DB.Column(EmailType, comment="The email address for which mails will be coming in")
+    recipient: Column = DB.Column(EmailType, comment="The email address to which mails will be going")
```
Comment (haslersn) [inline mail_alias_manager/db/models/receiving_alias.py]:
```suggestion
    alias: Column = DB.Column(EmailType, comment='The "To" address that should be redirected')
    recipient: Column = DB.Column(EmailType, comment='The "To" address to redirect to')
```

File: mail_alias_manager/db/models/receiving_alias.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.sql.schema import Column
+from sqlalchemy_utils.types.email import EmailType
+
+from ..db import DB, MODEL
+
+
+class ReceivingAlias(MODEL):
```
Comment (haslersn) [inline mail_alias_manager/db/models/receiving_alias.py]: Maybe call this `RecipientAlias`

```suggestion
class RecipientAlias(MODEL):
```

File: mail_alias_manager/db/models/sending_alias.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.sql.schema import Column
+from sqlalchemy_utils.types.email import EmailType
+
+from ..db import DB, MODEL
+
+
+class SendingAlias(MODEL):
```
Comment (haslersn) [inline mail_alias_manager/db/models/sending_alias.py]: Maybe call this `SenderAlias` or `SenderIdentity`. If you feel like `SenderIdentity` is better, then also rename the `alias` attribute to `identity`.

File: mail_alias_manager/db/models/sending_alias.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    id: Column = DB.Column(DB.Integer, primary_key=True)
+    sender: Column = DB.Column(DB.String(120), comment="The username of the sender who has this alias")
+    alias: Column = DB.Column(EmailType, comment="The email address for which mails should be able to be sent out")
```
Comment (haslersn) [inline mail_alias_manager/db/models/sending_alias.py]:
```suggestion
    alias: Column = DB.Column(EmailType, comment="The email address which the sender is allowed to impersonate")
```

**Review by haslersn — CHANGES_REQUESTED** [review]

File: mail_alias_manager/db/models/receiving_alias.py (hunk 2 above)
Comment (neumantm) [reply mail_alias_manager/db/models/receiving_alias.py]: I think ReceivingAlias is better

File: mail_alias_manager/db/models/sending_alias.py (hunk 3 above)
Comment (neumantm) [reply mail_alias_manager/db/models/sending_alias.py]: I think SendingAlias is better

File: mail_alias_manager/db/models/sending_alias.py (hunk 4 above)
Comment (neumantm) [reply mail_alias_manager/db/models/sending_alias.py]: "allowed to use" ?

File: mail_alias_manager/db/models/receiving_alias.py (hunk 1 above)
Comment (neumantm) [reply mail_alias_manager/db/models/receiving_alias.py]: ok

File: mail_alias_manager/db/models/receiving_alias.py (hunk 2 above)
Comment (haslersn) [reply mail_alias_manager/db/models/receiving_alias.py]: The idea was that we reduce the amount of different words used here. `RecipientAlias` is comprised of the same words used in the attributes.

File: mail_alias_manager/db/models/sending_alias.py (hunk 4 above)
Comment (haslersn) [reply mail_alias_manager/db/models/sending_alias.py]: Ok. Or "allowed to send as".

**Pull request merged**